build.gradle.kts

```diff
@@ -5,7 +5,7 @@ group = "no.nav.syfo"
 version = "0.0.1"
 
 object Versions {
-    const val confluent = "7.1.1"
+    const val confluent = "7.2.1"
```

Comment:
Tror kanskje Kafka bør oppgraderes til 3.2.0 med det samme for å holde ting konsistent.